app/controllers/users_controller.rb

```diff
@@ -1,5 +1,26 @@
 class UsersController < ApplicationController
+  attr_reader :user
+
   def new
     @user = User.new
   end
+
+  def show
+    @user = User.find(params[:id])
```

Comment:
không hẳn là where tham số 1 = đâu. toán tử gì thì còn tùy nhiều thứ khác nữa.
với lại có sự khác nhau quan trọng nữa cơ. tìm hiểu thêm xem